crates/rpc/src/types.rs

```diff
@@ -35,3 +46,15 @@ pub struct ProveBundleResponse {
     public_input: Bytes,
     signature: Signature,
 }
+
+sol! {
+    #[derive(Default, Serialize)]
+    struct ProveBundleSignatureData {
```

Comment:
shall we also add `layer2_chainId` here?

Reply:
Yeah, thanks, I missed this.